.github/workflows/release-please.yml

```
@@ -40,7 +40,7 @@ jobs:
         run: echo "::set-output name=dir::$(yarn config get cacheFolder)"
```

actionlint (1.7.4)

workflow command "set-output" was deprecated. use `echo "{name}={value}" >> $GITHUB_OUTPUT` instead: https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions

(deprecated-commands)